packages/nouns-webapp/src/components/Proposals/index.tsx

```diff
@@ -113,6 +113,171 @@ export interface SnapshotProposal {
   [key: string]: any;
 }
 
+export const LilNounProposalRow = ({ proposal }: { proposal: Proposal }) => {
```

Comment:
The changes in this file were just to refactor into smaller components/utilities so they can be exported and reused.